Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36121&attempt_id=c7af2519-d34e-4763-bd24-ba8fc89a3005)

<details><summary>Partial log (click to expand)</summary><p>

```
checking for miniupnpc/upnpcommands.h... yes
checking for main in -lminiupnpc... (cached) yes
checking miniupnpc/upnperrors.h usability... yes
checking miniupnpc/upnperrors.h presence... yes
checking for miniupnpc/upnperrors.h... yes
checking for main in -lminiupnpc... (cached) yes
checking for Qt5Core Qt5Gui Qt5Network Qt5Widgets... no
configure: error: Qt dependencies not found
builder for '/nix/store/l4l64vx9m952k6s3lkq6l89w8hga79qr-bitcoin-0.16.0.drv' failed with exit code 1
error: build of '/nix/store/l4l64vx9m952k6s3lkq6l89w8hga79qr-bitcoin-0.16.0.drv' failed
```
</p></details>

--- a/pkgs/applications/altcoins/bitcoin.nix
+++ b/pkgs/applications/altcoins/bitcoin.nix
@@ -36,6 +36,7 @@ stdenv.mkDerivation rec{
     homepage = http://www.bitcoin.org/;
     maintainers = with maintainers; [ roconnor AndersonTorres ];
     license = licenses.mit;
-    platforms = platforms.unix;
+    # bitcoin needs hexdump to build, which doesn't seem to build on darwin at the moment.
+    platforms = platforms.linux;
   };
 }

